src/variant_detection/variant_output.cpp

```diff
@@ -32,10 +32,11 @@ void find_and_output_variants(std::map<std::string, int32_t> & references_length
                     size_t insert_size = clusters[i].get_average_inserted_sequence_size();
                     if (mate1.orientation == strand::forward)
                     {
-                        int32_t distance = mate2_pos - mate1_pos;
+                        int32_t distance = mate2_pos - mate1_pos - 1;
+                        int32_t sv_length = insert_size - distance;
                         //Deletion
-                        if (distance >= args.min_var_length &&
-                            distance <= args.max_var_length &&
+                        if (sv_length <= -args.min_var_length &&
+                            sv_length >= -args.max_var_length &&
```

Comment:
These lines are very confusing... is sv_length a negative number?
I would suggest to write:
```suggestion
                        if (-sv_length >= args.min_var_length &&
                            -sv_length <= args.max_var_length &&
```